src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const errorMessage = 'Bzzz... Error!';
+
+  return (num1) => {
+    counter++;
+
+    const isMoreThen3 = counter > 3;
```

Comment:
It is necessary to choose a more meaningful name, for example, `isMaxOperationsCalled`.
And 3 is also "magic value".